e2e/js_image_oci/WORKSPACE

```diff
@@ -119,6 +119,88 @@ http_archive(
 
 load("@com_google_googleapis//:repository_rules.bzl", "switched_rules_by_language")
 
-switched_rules_by_language(
-    name = "com_google_googleapis_imports",
+switched_rules_by_language(name = "com_google_googleapis_imports")
+
+# LLVM
+http_archive(
```

Comment:
LLVM toolchain needed on MacOS for cross-compilation. I think this steps from the cc toolchain leak in the rules_nodejs toolchain. Fix for that tracked here: https://github.com/bazelbuild/rules_nodejs/issues/3722